Roadmap.md

```diff
@@ -0,0 +1,44 @@
+# karmada operator roadmap
+
+The karmada operator is a new project of karmada community designed to manage the lifecycle of karmada. The following is the core feature of the karmada operator that we will work on. It is not a detailed to-do list, but represents overall development direction. It is hoped that more contributors will have a general understanding of project and participate in it.
```

Comment:
all the `karmada` should change to `Karmada`.

Reply:
updated.